main.go

```diff
@@ -274,6 +284,62 @@ minio --version`, arch, semVerTag, debArchMap[arch], semVerTag, debArchMap[arch]
 					},
 				}
 
+				d.Subscriptions[subscription].Linux["AIStor Sidekick"][arch] = downloadJSON{
+					RPM: &dlInfo{
+						Download: fmt.Sprintf("https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick-%s-1.%s.rpm", arch, semVerTag, rpmArchMap[arch]),
+						Checksum: fmt.Sprintf("https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick-%s-1.%s.rpm.sha256sum", arch, semVerTag, rpmArchMap[arch]),
+						Text: fmt.Sprintf(`# Download the RPM package
+wget https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick-%s-1.%s.rpm
+
+# Install with yum/dnf
+sudo yum install sidekick-%s-1.%s.rpm
+# or
+sudo dnf install sidekick-%s-1.%s.rpm`, arch, semVerTag, rpmArchMap[arch], semVerTag, rpmArchMap[arch], semVerTag, rpmArchMap[arch]),
+					},
+					Deb: &dlInfo{
+						Download: fmt.Sprintf("https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick_%s_%s.deb", arch, semVerTag, debArchMap[arch]),
+						Checksum: fmt.Sprintf("https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick_%s_%s.deb.sha256sum", arch, semVerTag, debArchMap[arch]),
+						Text: fmt.Sprintf(`# Download the DEB package
+wget https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick_%s_%s.deb
+						
+# Install with apt
+sudo apt install ./sidekick_%s_%s.deb
+# or
+sudo dpkg -i sidekick_%s_%s.deb`, arch, semVerTag, debArchMap[arch], semVerTag, debArchMap[arch], semVerTag, debArchMap[arch]),
+					},
+					APK: &dlInfo{
+						Download: fmt.Sprintf("https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick_%s_%s.apk", arch, semVerTag, debArchMap[arch]),
+						Checksum: fmt.Sprintf("https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick_%s_%s.apk.sha256sum", arch, semVerTag, debArchMap[arch]),
+						Text: fmt.Sprintf(`# Download the APK package
+wget https://dl.min.io/aistor/sidekick/release/linux-%s/sidekick_%s_%s.apk
+
+# Install with apk
+sudo apk add --allow-untrusted sidekick_%s_%s.apk`, arch, semVerTag, apkArchMap[arch], semVerTag, apkArchMap[arch]),
+					},
```

Comment:
I think we can omit this as well, unless it is a requirement that we do it for APK format as well. This is being added only for sidekick, and for none of the other products (like mc or minio), so let's confirm on the requirement here.